src/crux/tx.clj

```diff
@@ -21,9 +21,8 @@
 
 (s/def ::id (s/conformer (comp str c/new-id)))
 (s/def :crux.db/id (s/and (complement string?) c/valid-id?))
-(s/def ::doc (s/and (s/or :doc (s/and (s/keys :req [:crux.db/id]) ::id)
-                          :content-hash (s/and (complement map?) ::id))
-                    (s/conformer second)))
+
+(s/def ::doc (s/and (s/keys :req [:crux.db/id]) ::id))
```

Comment:
This `s/and` can be removed.

Reply:
Taking this out brakes it so I think the s\and is needed.

Reply:
Yes, sorry, my mistake, the `s/and` is needed, confused it with the original `s/or`. The `::id` spec also calls `s/conformer` so the message should still contain the hash, so this change might actually be good. But worth sanity checking this first by printing out the message going onto the wire before we merge.

Reply:
Also, my checked, and there's currently no spec on the ingest side. This is a problem, but not necessarily part of this PR as that issue was already there, up to you and @jonpither.

Reply:
Actually, instead of just printing it out, you could simply add this to your test and see that the conformed message is indeed containing the hash and not the doc, as all data is already setup there to do this.